<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="IBS-LAP-324" failures="1" tests="7" name="API Test Cases" time="3.408" errors="0" timestamp="2025-09-17T11:13:03 IST">
  <testcase classname="com.api.test.AccountCreationTest" name="AccountCreationTesting" time="2.009"/>
  <testcase classname="com.api.test.ForgotPasswordTest" name="Test" time="2.031"/>
  <testcase classname="com.api.test.LoginAPITest2" name="loginTest" time="2.087"/>
  <testcase classname="com.api.test.getUserProfileManagementTest" name="Test" time="2.457"/>
  <testcase classname="com.api.test.UserProfileUpdateTest" name="userProfileUpdattionTest" time="0.679">
    <failure type="java.lang.AssertionError" message="expected [200] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at com.api.test.UserProfileUpdateTest.userProfileUpdattionTest(UserProfileUpdateTest.java:34)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- userProfileUpdattionTest -->
  <testcase classname="com.api.test.patchUpdateProfileTest" name="updateProfileTest" time="0.715"/>
  <testcase classname="com.api.test.deleteUserProfileTest" name="deleteUserProfile" time="3.355"/>
</testsuite> <!-- API Test Cases -->
